assets/src/less/blocks/modifiers/theme/has-x-color.less

```diff
@@ -0,0 +1,13 @@
+/**
+ * Has X Color
+ */
+
+.has-text-color,
+.has-inline-color {
+	a:not([class]) {
```

Comment:
Ah @douglas-johnson This `:not( [class] )` selector rule is cool! I haven't used this before.